server/serverRun.py

```diff
@@ -135,60 +145,67 @@ def login():
 
```

Comment:
This is a great change. I was working on this in parallel because I noticed that even if we were unauthorized, it's was returning 200 OK.

Reply:
That's in another branch though. This'll be nice to rebase off of.